### DISCUSSION TITLE
Why the source code doesn't reflect new releases?

### DISCUSSION THREAD

**Discussion by rwp0** [StrawberryPerl/Perl-Dist-Strawberry#111] [Q&A] [answered]
The `master` branch last updated Apr 21, while we see new releases today.

How can we view the source code for the new Strawberry releases?

Is it in another branch, or not kept in this repository (ie. public)?

**Comment by shawnlaffan** [comment, marked as answer]
It's on the wip_536 branch.  

I'll set up a PR once I have cleaned up the gdb component and then made similar changes to the 5.36 config file.

**Comment by rwp0** [comment]
Oh, I thought that would cover v5.36 work only:

https://github.com/StrawberryPerl/Perl-Dist-Strawberry/tree/wip_536

**Comment by shawnlaffan** [comment]
The branch name has become somewhat out of date since Perl 5.38 RC1.

**Comment by shawnlaffan** [comment]
Each release is also linked with a tag.  This is usually to the left of the release details, although that might change if you are using a tablet or phone.

**Comment by shawnlaffan** [comment]
Main branch is now up to date.
